apps/webapp/src/components/header/constants.tsx

```diff
@@ -33,12 +32,6 @@ export const headerLinks: HeaderLink[] = [
     active: true,
     mobileIcon: <SwapIcon />,
   },
-  {
-    href: PagePath.POOLS,
-    label: 'Pools',
-    active: false,
-    mobileIcon: <DragHandleDotsIcon />,
-  },
```

Comment:
minifront won't be supporting liquidity provisioning

Reply:
What's minifront? 😬

Reply:
"mini frontend" is what we wanted to rename "web app" https://discord.com/channels/824484045370818580/1044457038886998137/1200144643321516032